xml/System.Collections.Generic/Dictionary`2.xml

```diff
@@ -2778,11 +2778,11 @@ finally {
         <Parameter Name="value" Type="TValue" Index="1" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="key">The key to be added to the dictionary.</param>
+        <param name="value">The value to be added to the dictionary.</param>
+        <summary>Attempts to add a key/value to the dictionary, returning <see langword="true" /> if added and <see langword="false" /> otherwise.</summary>
+        <returns><see langword="true" /> if the key was added successfully, and <see langword="false" /> otherwise.</returns>
+        <remarks>Unlike the <xref:System.Collections.Generic.Dictionary%602.Add%2A> method, this method does not throw an exception if the element with the given key exists in the dictionary. Unlike the Dictionary indexer, TryAdd does not override the element if the element with the given key exists in the dictionary. If the key already exists, TryAdd does nothing and returns <see langword="false" />.</remarks>
```

Comment:
❔ Not sure if I need to add refs for the `key` parameter -- I can if it'd be helpful.